src/pages/index.js

```diff
@@ -75,51 +136,95 @@ const profileAddPopup = new PopupWithForm(
   handleAddFormSubmit
 );
 
+const profileAvatarPopup = new PopupWithForm(
+  "#profile-avatar-modal",
+  handleAvatarFormSubmit
+);
+
 const profilePreviewPopup = new PopupWithImage("#profile-preview-modal");
+const confirmationPopup = new PopupWithConfirmation("#confirmation-modal");
 
 profileEditPopup.setEventListeners();
 profileAddPopup.setEventListeners();
+profileAvatarPopup.setEventListeners();
 profilePreviewPopup.setEventListeners();
+confirmationPopup.setEventListeners();
 
 // SECTION INIT
 const cardSection = new Section(
   {
-    items: initialCards,
+    items: [],
     renderer: (item) => {
-      const cardElement = createCard(item);
+      const cardElement = createCard(item, currentUserId);
       cardSection.addItem(cardElement, true);
```

Comment:
` true` is not needed here because the initial cards should be added with `append`